ttnn/cpp/ttnn/config.hpp

```diff
@@ -90,6 +91,18 @@ struct Config {
             }
         }
 
+        if (
```

Comment:
could we clean up this nested condition structure? maybe early return if logging is disabled at the top of the function?